core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -139,6 +150,15 @@ public S3RestServiceHandler(@Context ServletContext context) {
     mBucketAdjacentDotsDashesPattern = Pattern.compile("([-\\.]{2})");
     mBucketInvalidPrefixPattern = Pattern.compile("^xn--.*");
     mBucketInvalidSuffixPattern = Pattern.compile(".*-s3alias$");
+
+    if (Configuration.getBoolean(PropertyKey.PROXY_AUDIT_LOGGING_ENABLED)) {
+      mAsyncAuditLogWriter = new AsyncUserAccessAuditLogWriter("PROXY_AUDIT_LOG");
+      mAsyncAuditLogWriter.start();
+      MetricsSystem.registerGaugeIfAbsent(
+          MetricKey.MASTER_AUDIT_LOG_ENTRIES_SIZE.getName(),
```

Comment:
Should we define a new metric key for the Proxy audit log or are we fine with just copying the value from the master?